lib/web_ui/lib/window.dart

```diff
@@ -29,118 +29,84 @@ abstract class FlutterView {
 }
 
 abstract class SingletonFlutterWindow extends FlutterView {
```

Comment:
Note: this should be marked as deprecated:

https://github.com/flutter/engine/blob/9cb8599d81669603f0b226a56c25fcd6259b20ed/lib/ui/window.dart#L406-L411